app/src/main/res/layout/nav_section_filter_divider.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="vertical">
+
+    <View
+        android:layout_width="match_parent"
+        android:layout_height="1dp"
+        android:layout_centerVertical="true"
+        android:background="?android:attr/listDivider"
+        tools:background="@android:color/holo_red_dark"/>
+
+    <LinearLayout
+        android:id="@+id/nav_filter_message"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_alignParentTop="true"
+        android:background="?attr/selectableItemBackground"
+        android:contentDescription="@string/filtered_label"
+        android:focusable="true"
```

Comment:
Remove this line to fix clicking the items

Reply:
Thank you very much. This saved my day :+1: